src/test_utils/hive/mod.rs

```diff
@@ -0,0 +1,180 @@
+use std::collections::HashMap;
+
+use ef_testing::evm_sequencer::account::{AccountType, KakarotAccount};
+use ethers::types::U256 as EthersU256;
+use eyre::eyre;
+use katana_primitives::{
+    contract::ContractAddress,
+    genesis::json::{GenesisContractJson, GenesisJson},
+};
+use reth_primitives::{Address, Bytes, B256, U256, U64};
+use serde::{Deserialize, Serialize};
+use starknet::core::utils::get_storage_var_address;
+use starknet_api::core::ClassHash;
+use starknet_crypto::FieldElement;
+
+use super::katana::genesis::{KatanaGenesisBuilder, Loaded};
+
+/// Types from https://github.com/ethereum/go-ethereum/blob/master/core/genesis.go#L49C1-L58
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct HiveGenesisConfig {
+    pub config: Config,
+    pub coinbase: Address,
+    pub difficulty: U64,
```

Comment:
how do we handle the difficulty variable? since ours is hardcoded to 0?

Reply:
not handled